README.md

```diff
@@ -24,34 +24,37 @@ npm install -g contentful-export
 Usage: contentful-export [options]
 
 Options:
+  --version             Show version number                            [boolean]
 
---version             Show version number [boolean]
+  --space-id            ID of Space with source data         [string] [required]
 
---space-id            ID of Space with source data [string] [required]
+  --management-token    Management API token for the space to be exported.
```

Comment:
It might be a bit redundant but I would add a `Contentful` infront of `Management API token ...` -> `Contentful Management API token...`